dynamicdns.go

```diff
@@ -385,30 +377,30 @@ func (a App) allDomains() map[string][]string {
 }
 
 // recordType returns the DNS record type associated with the version of ip.
-func recordType(ip net.IP) string {
-	if ip.To4() == nil {
-		return recordTypeAAAA
+func recordType(addr netip.Addr) string {
+	if addr.Is4() {
+		return recordTypeA
 	}
-	return recordTypeA
+	return recordTypeAAAA
```

Comment:
This makes me wonder if we should export a method like this on the `libdns.Address` type.

Reply:
Not necessary, you just do `addr.IP.Is4()` (i.e. the IP field on the libdns.Address type)

Reply:
I guess that's true. It's just inline-able this way, but good point. If I see this function pop up more I might just make it available in libdns anyway. :upside_down_face:

Reply:
This function should probably be removed after additional refactoring.  It's only left since there is a map of domains to record types to a list of IPs.  The record types themselves don't seem to actually matter, it's just used to separate v4 and v6.